src/pages/cli/auth/admin.mdx

```diff
@@ -263,4 +263,52 @@ class MyCustomInterceptor: URLRequestInterceptor {
 }
 ```
 </Block>
-</BlockSwitcher>
\ No newline at end of file
+</BlockSwitcher>
+
+## Adding Admin Actions
+
+To add additional admin actions that are not included by default but are enabled by Amazon Cognito, you will need to update the Lambda function code that is generated for you. The change will include adding a procedure for the action and creating a route for it. You will then associate the procedure to the route within the express app.
+
+Below is an example of how to add an admin action that will allow you to <ExternalLink href="https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminUpdateUserAttributes.html">update a user's attributes</ExternalLink>.
```

Comment:
```suggestion
Below is an example of how to add an admin action that will allow you to [update a user's attributes](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminUpdateUserAttributes.html).
```

minor nit for link usage in markdown

https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#20-write-links-using-markdown-syntax

Reply:
Since the link is going to docs.aws.amazon.com and not docs.amplify.aws is it not considered an "External link"? Or are external link classified as non-AWS?